### PR TITLE
Add explicit okhttp3-logging-interceptor

### PR DESCRIPTION
**Pull request by RotBolt**
cc @michalszynkiewicz

### CODE REVIEW

File: bom/application/pom.xml [hunk 1]
```diff
@@ -165,6 +165,7 @@
         <artemis.version>2.11.0</artemis.version>
         <proton-j.version>0.33.6</proton-j.version>
         <okhttp.version>3.14.9</okhttp.version>
+        <okhttp-logging.version>3.14.3</okhttp-logging.version>
```
Comment (michalszynkiewicz) [inline bom/application/pom.xml:168]: maybe rename it to okhttp3.version?

Comment (RotBolt) [reply bom/application/pom.xml:168]: both versions are okhttp3 actually. Its logging specific. thats why I added logging

**Review by gsmet — CHANGES_REQUESTED** [review]
So... either it's a quick fix for 1.7 and it doesn't need to be applied to the other branches (which I doubt) or you need to create the PR against master and add the `triage/backport-1.7?` label.

Otherwise we will miss it next time.

Thanks.

File: bom/application/pom.xml (hunk 1 above)
Comment (michalszynkiewicz) [reply bom/application/pom.xml:168]: the groupId for okhttp thing is different, isn't it?

Comment (gsmet) [reply bom/application/pom.xml:168]: Is there a reason why we can't use the same version? I see https://repo1.maven.org/maven2/com/squareup/okhttp3/logging-interceptor/3.14.9/ ?

If there is a good reason, let's add a comment. If there is not, let's use the same version and have only one property.

Comment (gsmet) [reply bom/application/pom.xml:168]: @michalszynkiewicz no it's exactly the same. That's why I think there's something fishy with defining 2 inconsistent versions.

Comment (RotBolt) [reply bom/application/pom.xml:168]: > the groupId for okhttp thing is different, isn't it?
artifact is different only

Comment (michalszynkiewicz) [reply bom/application/pom.xml:168]: oh, it's my mistake, I took another look at our analysis file and we indeed use 3.14.9 only for other stuff. @RotBolt could you update this to use okhttp.version for both dependencies?

Comment (RotBolt) [reply bom/application/pom.xml:168]: Ok sure updating it

**Comment by RotBolt** [comment]
@michalszynkiewicz I guess now I should make PR for master and put lable triage/backport ?

**Comment by michalszynkiewicz** [comment]
> @michalszynkiewicz I guess now I should make PR for master and put lable triage/backport ?

Yes please :)

**Pull request closed without merging**